tests/test_backend.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 from kudos.backend import UserBackend, DuplicateEmailError
+from kudos.backend import InvalidEmailFormatError
```

Comment:
Put all imports from the same file on the same line, also you can break the imports into multiple lines using brackets:

'''
from file import (
cls1,
cls2,
cls3,
)
'''